src/styles/header.scss

```diff
@@ -0,0 +1,104 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Use Roboto font */
+.body-item {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+/* Header styling */
+.header-item {
+  background-color: #fff;
+  justify-content: space-between;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+/* Logo styling */
+.logo img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+/* Navigation styling */
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+  justify-content: center; /* Центрування по горизонталі */
+  align-items: center; /* Центрування по вертикалі */
```

Comment:
The comments in these lines are not in English. It's a good practice to keep comments in English to maintain consistency and readability across the codebase.